README.md

```diff
@@ -110,6 +110,41 @@ glfs: /volume@hostname/filename
 5. The created backstore device can then be mapped to a LUN like traditional
 backstores.
 
+##### Logger setting and system configuration
+
+- Logger setting:
+
+There are 5 logging levels supported:
+
+1. ERROR
+2. WARNING
+3. INFO
+4. DEBUG
+5. DEBUG SCSI CMD
+
+And the default logging level is 3, if you want to change the default level,
+uncomment the following line in /etc/tcmu/tcmu.conf and set your level number:
+
+\# log_level = 3
+
+The log file's location could be configurd in /etc/tcmu/tcmu.con too.
+
+The priority of the logdir setting is:
+
+1. --tcmu_log_dir/-l `LOG_DIR_PATH` [Highest]
+2. export TCMU_LOGDIR="/var/log/mychoice/"
+3. /etc/tcmu/tcmu.conf
+4. default /var/log/ as hard coded [Least]
+
```

Comment:
@lxbsz  Just a cosmic change, how about:

The priority of the logdir setting can be managed via following options:
1. Cli argument
     eg: --tcmu_log_dir/-l `LOG_DIR_PATH` [Highest prio]
2. Environment variable
    eg: export TCMU_LOGDIR="/var/log/mylogdir/"
3. Configuration file
     eg: uncommenting and adjusting value of 'log_dir_path' at  /etc/tcmu/tcmu.conf
4. Default logdir as hard coded i.e. '/var/log/' [Least prio]

IMO, doesn't need a re-spin, just for this :-)

Reply:
Looks much better than mine :-)